Lib/idlelib/idle_test/test_config.py

```diff
@@ -26,6 +28,162 @@ def tearDownModule():
     idleConf.userCfg = usercfg
 
 
+class IdleConfParserTest(unittest.TestCase):
+    """Test IdleConfParser works"""
+
+    config = """
+        [one]
+        one = false
+        two = true
+        three = 10
+
+        [two]
+        one = a string
+        two = true
+        three = false
+    """
+
+    def test_get(self):
+        parser = config.IdleConfParser('')
+        parser.read_string(self.config)
+
+        eq = self.assertEqual
+
+        # Test with type
+        eq(parser.Get('one', 'one', type='bool'), False)
```

Comment:
It is better to use `assertIs()`. Since `False` and `True` are singletons, this tests both type and value. Otherwise `parser.Get()` returning `0` will pass the test.